pkg/api/k8s_versions.go

```diff
@@ -227,76 +227,80 @@ func getK8sVersionComponents(version string, overrides map[string]string) map[st
 	switch majorMinor {
 	case "1.18":
 		ret = map[string]string{
-			"kube-apiserver":                              "kube-apiserver:v" + version,
```

Comment:
What specifically changed WRT 1.18 component versions? This diff is too hard to decipher since `gofmt` aligned everything.